modin/pandas/dataframe.py

```diff
@@ -1278,7 +1278,7 @@ def drop_duplicates(self, subset=None, keep="first", inplace=False):
         inplace = validate_bool_kwarg(inplace, "inplace")
         duplicates = self.duplicated(subset=subset, keep=keep)
         indices, = duplicates.nonzero()
```

Comment:
It looks like that `nonzero` is also deprecated in pandas 0.24. Could we change it to 
```suggestion
        indices, = duplicates.to_numpy().nonzero()
```
This also might be a good first issue to at least avoid using `nonzero` and, possibly, avoid `duplicated`

Reply:
Unfortunately, `to_numpy` is not available as of 0.23.4. We can't do that change yet.

Reply:
We can use `values` in the meantime.